spec.md

```diff
@@ -1407,6 +1518,13 @@ can be mounted on all app instances simultaneously.
     "host": "mysqlhost",
     "port": 3306,
     "database": "dbname"
+  },
+  "endpoints": [
+    "mysqlhost:3306"
```

Comment:
Following the proposal to use a struct instead of a concatenated string this would be: `{ "host": "mysqlhost", "port": 3306 }`